testing/tests/DevExpress.animation/position.tests.js

```diff
@@ -955,6 +956,64 @@ const testCollision = (name, fixtureName, params, expectedHorzDist, expectedVert
         }
     });
 
+    QUnit.test('position should return window.height() if window.outerHeight == window.innerHeight (T939748)', function(assert) {
+        const isPhone = devices.real().deviceType === 'phone';
+        if(isPhone || browser.safari) {
+            assert.ok(true, 'actual only for desktop browsers except Safari');
+            return;
+        }
+
+        const $what = $('#what').height(300);
+        const initialInnerHeight = window.innerHeight;
+        const initialOuterHeight = window.outerHeight;
+
+        const heightStub = sinon.stub(renderer.fn, 'height').returns(1000);
+
+        try {
+            window.innerHeight = 500;
```

Comment:
I think it's better to use a mock for window instead of modifying a real one.

Reply:
How can we do it? `innerHeight` is property, not method, so we can not use `sinon.stub`